regression_and_census.R

```diff
@@ -9,9 +9,166 @@
 # old code-  I M P O R T I N G   C E N S U S   D A T A ---------------------------------------
 
 library(latticeExtra)
-install.packages("gee")
+library(gee)
+library(geepack)
+library(MASS)
+library(broom)
 # tidyverse
 library(dplyr)
+library(readr)
+library(tidyr)
+library(magrittr)
+library(stringr)
+library(purrr)
+# plotting libraries
+library(ggplot2)
+library(ggmap)
+library(gridExtra)
+
+
+
+# Load data ---------------------------------------------------------------
+
+clean_weekly_data_dir <- "." #"/Users/funmiarogbokun/Desktop/CZ Biohub Contractor/Wildfires & Covid Project/Manuscript/Clean_final_datasets/Weekly_data"
+CA_PM25_wk_PM <- read_csv(file.path(clean_weekly_data_dir, "CA_weekly_lagged_averages.csv")) %>%
+  mutate(county=factor(county))
+
+census_data_dir <- "." #"/Users/funmiarogbokun/Desktop/CZ Biohub Contractor/Wildfires & Covid Project/Census"
+census_counties_2020 <- read_csv(file.path(census_data_dir, "census_CAcountydata_2020.csv"))
+
+
+# Clean and transform data for California ---------------------------------
+
+#merging the two datasets
+pol_covid_census_2020 <- CA_PM25_wk_PM %>%
```

Comment:
See above comment about using tidyverse if possible. `left_join` is the tidy verse equivalent of the `merge` function you had used before. 
Also, by using `by=c("county"="BASENAME")` in the argument, you can circumvent the need to create a new data frame just to rename the column.